src/scenes/DeliverablesRouter/QuestionsDeliverableView.tsx

```diff
@@ -162,12 +163,13 @@ const QuestionsDeliverableView = (props: Props): JSX.Element => {
   }, [deliverable]);
 
   useEffect(() => {
-    if (!currentParticipantProject?.id) {
+    if (!(deliverableId && projectId)) {
       return;
     }
 
-    void dispatch(requestListParticipantProjectSpecies(projectId));
-  }, []);
+    void dispatch(requestListDeliverableVariables(deliverableId));
```

Comment:
Since both of these are using "argument as key", we don't need to keep track of the request.